hooks/tk-multi-publish2/tk-maya/basic/publish_turntable.py

```diff
@@ -970,15 +980,22 @@ def publish(self, settings, item):
         publish_path = self.get_publish_path(settings, item)
         publish_path = os.path.normpath(publish_path)
 
-        # This plugin publishes a turntable movie to Shotgun
+        # This plugin publishes a turntable movie to SG
         # These are the steps needed to do that
 
         # =======================
         # 1. Export the Maya scene to FBX
         # The FBX will be exported to a temp folder
         # Another folder can be specified as long as the name has no spaces
         # Spaces are not allowed in command line Unreal Python args
-        temp_folder = tempfile.mkdtemp(suffix="temp_unreal_shotgun")
+
+        # Set a base temp dir on Windows to avoid having
+        # the user name in the temp path which can include
+        # "." e.g. firstname.name and makes UE crashes
+        base_temp_dir = None
+        if sys.platform == "win32":
+            base_temp_dir = r"C:\Temp"
```

Comment:
Is it safe to assume this directory exists on the filesystem?

Reply:
Not 100% sure actually. But I didn't create this directory myself, and it is referenced in [this](https://www.autodesk.com/support/technical/article/caas/sfdcarticles/sfdcarticles/How-to-delete-temporary-files-in-Windows.html)

Reply:
Sounds fair.